modules/express/src/clientRoutes.ts

```diff
@@ -453,6 +453,20 @@ async function handleV2ConsolidateUnspents(req: express.Request) {
   return wallet.consolidateUnspents(req.body);
 }
 
+/**
+ * handle wallet fanout unspents
+ * @param req
+ */
+async function handleV2ConsolidateAccount(req: express.Request) {
+  const bitgo = req.bitgo;
+  const coin = bitgo.coin(req.params.coin);
+  if (coin.getFamily() !== 'hbar') {
+    throw new Error('invalid coin selected');
+  }
+  const wallet = await coin.wallets().get({ id: req.params.id });
+  return wallet.sendAccountConsolidations(req.body);
```

Comment:
Please check wallet return value before use, otherwise you'll emit an unhelpful error to the caller.